exporter/opentelemetry-exporter-jaeger-proto/src/opentelemetry/exporter/jaeger/proto/__init__.py

```diff
@@ -0,0 +1,176 @@
+    PostSpansRequest,
+)
+from opentelemetry.exporter.jaeger.proto.gen.collector_pb2_grpc import (
+    CollectorServiceStub,
+)
+from opentelemetry.exporter.jaeger.proto.translate import (
+    ProtobufTranslator,
+    Translate,
+)
+from opentelemetry.sdk.environment_variables import (
+    OTEL_EXPORTER_JAEGER_ENDPOINT,
+)
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
+
+DEFAULT_GRPC_COLLECTOR_ENDPOINT = "localhost:14250"
+
+logger = logging.getLogger(__name__)
+
+
+class JaegerExporter(SpanExporter):
+    """Jaeger span exporter for OpenTelemetry.
+
+    Args:
+        collector_endpoint: The endpoint of the Jaeger collector that uses
+            Protobuf via gRPC.
+        insecure: True if collector has no encryption or authentication
+        credentials: Credentials for server authentication.
+        max_tag_value_length: Max length string attribute values can have. Set to None to disable.
+    """
+
+    def __init__(
+        self,
+        collector_endpoint: Optional[str] = None,
+        insecure: Optional[bool] = None,
+        credentials: Optional[ChannelCredentials] = None,
+        max_tag_value_length: Optional[int] = None,
+    ):
+        self._max_tag_value_length = max_tag_value_length
+
+        self.collector_endpoint = _parameter_setter(
+            param=collector_endpoint,
+            env_variable=environ.get(OTEL_EXPORTER_JAEGER_ENDPOINT),
+            default=None,
+        )
+        self._grpc_client = None
+        self.insecure = insecure
+        self.credentials = util._get_credentials(credentials)
+        tracer_provider = trace.get_tracer_provider()
+        self.service_name = (
+            tracer_provider.resource.attributes[SERVICE_NAME]
+            if getattr(tracer_provider, "resource", None)
+            else Resource.create().attributes.get(SERVICE_NAME)
+        )
+
+    @property
+    def _collector_grpc_client(self) -> Optional[CollectorServiceStub]:
+        endpoint = self.collector_endpoint or DEFAULT_GRPC_COLLECTOR_ENDPOINT
+
+        if self._grpc_client is None:
+            if self.insecure:
+                self._grpc_client = CollectorServiceStub(
+                    insecure_channel(endpoint)
+                )
+            else:
+                self._grpc_client = CollectorServiceStub(
+                    secure_channel(endpoint, self.credentials)
+                )
+        return self._grpc_client
+
+    def export(self, spans) -> SpanExportResult:
+        translator = Translate(spans)
+        pb_translator = ProtobufTranslator(
+            self.service_name, self._max_tag_value_length
+        )
+        jaeger_spans = translator._translate(pb_translator)
+        batch = model_pb2.Batch(spans=jaeger_spans)
+        request = PostSpansRequest(batch=batch)
+        self._collector_grpc_client.PostSpans(request)
+
+        return SpanExportResult.SUCCESS
```

Comment:
Out of scope for this PR. Probably we should follow up in another issue/PR. This seems to send `SpanExportResult.SUCCESS` status always; we should handle failures and return the correct export status.